Examples/SmokeTest/SmokeTest/ContentView.swift

```diff
@@ -65,6 +65,15 @@ struct ContentView: View {
             NetworkView()
                 .padding()
                 .tabItem { Label("Network", systemImage: "network") }
+
+            UIKView()
```

Comment:
The name `UIKView` feels very wrong to me. Objective C didn't have namespaces, so the convention was to use two letter (and later, three letter) prefixes to avoid naming collisions. "UI" was the prefix for UIKit. My brain can't help but read `UIKView` as a `View` in the `UIK` namespace. I'd recommend just making it `UIKitView`.